docker/package/Dockerfile-ubuntu-test

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2021 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
```

Comment:
Tests are not related to the packaging logic in any way, so they should be located in it's own directory. I propose `docker/tests` directory.